docs/data-model.md

```diff
@@ -0,0 +1,78 @@
+| A batch PR is submitted for this and other terms (for import and creation)                                                                                                   | awaiting_import      | reviewed                                                        | awaiting_import                                                  |
+| A term import is completed — the new ontology term status is caught automatically during a periodic OLS status refresh. <br>After import the same term will now be "current" | current              | reviewed                                                        | current                                                          |
+|                                                                                                                                                                              |                      |                                                                 |                                                                  |
+| **Case 3. Curator chooses a new term which is not in _any_ ontology (needs creation)**                                                                                       | needs_creation       | not_reviewed                                                    | awaiting_review                                                  |
+| necessary number of reviews is accumulated                                                                                                                                   | needs_creation       | reviewed                                                        | needs_creation                                                   |
+| A batch PR is submitted for this and other terms (for import and creation)                                                                                                   | awaiting_creation    | reviewed                                                        | awaiting_creation                                                |
+| After the PR is closed and new terms are created, the curator <br>manually selects a new term                                                                                | current              | not_reviewed                                                    | awaiting_review                                                  |
+| A necessary number of reviews for the newly created term is accumulated                                                                                                      | current              | reviewed                                                        | current                                                          |
+
+### Diagram:
+
+![](https://i.imgur.com/tUBBipu.png)
```

Comment:
It's better to commit the PNG directly into repository as a binary file than to rely on an external resource (even though, admittedly, Imgur is much better than hotlinking Mermaid servers directly).

While normally committing binary files (especially large) is bad, for documentation PNGs it's perfectly fine.